### PR TITLE
feat(hacbs-1414): change pac metadata prefix

### PR DESCRIPTION
**Pull request by parasense**
Resolves: hacbs-1414
Related: hacbs-1385

Signed-off-by: Jon Disnard <jdisnard@redhat.com>

### CODE REVIEW

File: tekton/pipeline_run.go
Comment (scoheb) [inline tekton/pipeline_run.go:53]: Why **test**?

Comment (johnbieren) [reply tekton/pipeline_run.go:53]: replying on slack

**Comment by johnbieren** [comment]
I've added the do-not-merge label as we shouldn't merge this until https://github.com/redhat-appstudio/integration-service/pull/69 merges

**Comment by johnbieren** [comment]
@parasense the tests are flaky right now. If you click details on the check sources check, in the top right you can click the circular arrows button to rerun it (without having to amend this PR). I've been doing that until I get it to pass, in order to get by the flakiness

File: tekton/pipeline_run.go
Comment (parasense) [reply tekton/pipeline_run.go:53]: Quoting from [hacbs-1385](https://issues.redhat.com/browse/HACBS-1385):

> Acceptance Criteria
>
>The integration controller replaces the prefix ("pipelinesascode.tekton.dev/") on the PaC labels with the AppStudio test prefix ("pac.test.appstudio.openshift.io/") when adding them to the Integration PipelineRun.

The **test** string is also reflected in the integration-service PR:
https://github.com/redhat-appstudio/integration-service/pull/69

**Review by johnbieren — CHANGES_REQUESTED** [review]
Now that the integration-service PR changed, I think we should just import the constant from their service (which won't work in an IDE until they are merged)

File: tekton/pipeline_run.go
Comment (johnbieren) [inline tekton/pipeline_run.go:54]: I think it would be better to not define this constant here and instead use `integrationServiceGitopsPkg.PipelineAsCodePrefix` everywhere `pipelinesAsCodeMetadataPrefix` is currently used

Comment (parasense) [reply tekton/pipeline_run.go]: That works great for me. Thanks.

**Review by johnbieren — APPROVED** [review]
lgtm once the CI passes

**Pull request merged**
